models/labels/airdrop/persona/optimism_rounds/labels_airdrop_1_receivers_optimism.sql

```diff
@@ -3,12 +3,12 @@
 SELECT
     'optimism' as blockchain,
     address,
-    '$OP Airdrop 1 Receiver' AS name,
+    'OP Airdrop 1 Receiver' AS name,
     'airdrop' AS category,
     'soispoke' AS contributor,
     'query' AS source,
     timestamp('2022-09-29') as created_at,
     now() as updated_at,
     'op_airdrop_1_receivers' AS model_name,
     'persona' as label_type
-FROM {{ ref('airdrop_optimism_addresses_1') }}
\ No newline at end of file
+FROM {{ ref('airdrop_optimism_addresses_1') }}
```

Comment:
Hi @MSilb7, please review the `labels` part to check if I make any mistake during resolving conflicts.

Reply:
hey seems good. It's just this ref name right"

Reply:
yes